packages/web-components/jest.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   },
   transform: {
     '^.+\\.svg$': '<rootDir>/../../scripts/jest/svg-transform.js',
-    '^.+\\.svelte$': 'svelte-jester',
+    '^.+\\.svelte$': ["svelte-jester", { "preprocess": true }],
```

Comment:
I removed the pre-processor as it makes your text editor slow. Also, it breaks jest!